src/Codeception/Module/FTP.php

```diff
@@ -82,7 +82,19 @@
  *              key: ''
  *              tmp: 'tests/_data/ftp'
  *              cleanup: false
+ * *  *  * #### Example (SFTP) for symphony
```

Comment:
what is this symphony thing?

Reply:
framework symfony

Reply:
how is FTP module related to symfony framework?

Reply:
i use codeception with this framework

Reply:
I agree, symfony or symphony are not related here :)